aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -114,6 +116,16 @@ module AptosFramework::Token {
         maximum: Option<u64>,
     }
 
+    /// The royalty of a token
+    struct Royalty has copy, drop, store {
+        // The basis point and denominator is ROYAL_BASIS_POINT_DENOMINATOR (1,000,000)
+        royalty_points_per_million: u64,
```

Comment:
it might be better to have two values rather than a single value much like Coin: https://github.com/aptos-labs/aptos-core/blob/d16d8572a473a521533db0eb519895b68027efd4/aptos-move/framework/aptos-framework/sources/Coin.move#L67

Reply:
Not sure if you have seen the reply in other PR 
"
decimal in coin is for getting a user representation of coin amount. Here, we need both nominator and denominator so that we can represent a percentage. If you want the denominator to be flexible. we can make it a variable entered by the users. However, it is better to keep the denominator a constant across different tokens. So the royal points (nominator) are comparable across different tokens, eg: Metaplex assuming the denominator is 100 in code"